src/components/ReplyForm/ReplyButton.tsx

```diff
@@ -1,22 +1,34 @@
 import React, { HTMLAttributes } from 'react';
 import classNames from 'classnames';
+import noop from 'lodash/noop';
 
 export type Props = {
     children?: React.ReactNode;
     isDisabled?: boolean;
     isPrimary?: boolean;
+    onClick?: (event: React.MouseEvent) => void;
     type: 'button' | 'submit' | 'reset' | undefined;
 } & HTMLAttributes<HTMLButtonElement>;
 
-export default function ReplyButton({ children, isDisabled, isPrimary, ...rest }: Props): JSX.Element {
+export default function ReplyButton({ children, isDisabled, isPrimary, onClick = noop, ...rest }: Props): JSX.Element {
+    const handleClick = (event: React.MouseEvent) => {
+        if (isDisabled) {
+            event.preventDefault();
+            event.stopPropagation();
+            return;
+        }
+
+        onClick(event);
+    };
+
     return (
         <button
             aria-disabled={isDisabled ? true : undefined}
             className={classNames('btn', {
                 'btn-primary': isPrimary,
                 'is-disabled': isDisabled,
             })}
-            disabled={isDisabled ? true : undefined}
+            onClick={handleClick}
```

Comment:
Is it bad if we leave the `disabled` attribute on the button? Seems semantically correct and then we can avoid the extra `handleClick` logic to manually implement a disabled button

Reply:
I agree, which is why I built it that way originally. I'm just following the behavior in BUIE, though.

Reply:
👍